demos/fonts/with-bravura.html

```diff
@@ -0,0 +1,26 @@
+<!DOCTYPE html>
+<!-- 
+  vexflow-core-with-bravura.js bundles the Bravura music font. 
+  There is no need to call Vex.Flow.setMusicFont(...).
+-->
+<html>
+  <body>
+    <div id="output"></div>
+    <script src="../../build/vexflow-core-with-bravura.js"></script>
```

Comment:
These are new! Simon of OSMD requested support for smaller builds without undesired fonts.

`vexflow-core.js` bundles zero fonts. Load fonts at runtime by calling `Flow.setMusicFont(...fontNames)`.

`vexflow-core-with-bravura.js` bundles only the Bravura font by default. Other fonts can be dynamically loaded with a call to `Flow.setMusicFont(...fontNames)`.